src/safeds_runner/server/main.py

```diff
@@ -14,15 +14,6 @@ def start_server(port: int) -> None:
     builtins.print = functools.partial(print, flush=True)  # type: ignore[assignment]
 
     logging.getLogger().setLevel(logging.DEBUG)
-    # Startup early, so our multiprocessing setup works
-    app_pipeline_manager = PipelineManager()
-    app_pipeline_manager.startup()
-    from gevent.monkey import patch_all
 
-    # Patch WebSockets to work in parallel
-    patch_all()
-
-    from safeds_runner.server.server import SafeDsServer
-
-    safeds_server = SafeDsServer(app_pipeline_manager)  # pragma: no cover
+    safeds_server = SafeDsServer()  # pragma: no cover
     safeds_server.listen(port)  # pragma: no cover
```

Comment:
Is the `# pragma: no cover` really needed here? It seems like if line 16 is run, so are the subsequent lines.

Reply:
I agree, line 18 should be covered. However, line 19 does never return, so it should stay that way. 
- If successful, the eventloop runs forever
- If unsuccessful, the stack unwinds
- If a shutdown was requested, the runner exits via a syscall